blocks_vertical/control.js

```diff
@@ -206,11 +201,10 @@ Blockly.Blocks['control_stop'] = {
     this.appendDummyInput()
         .appendField(Blockly.Msg.CONTROL_STOP)
         .appendField(stopDropdown, 'STOP_OPTION');
-    this.setCategory(Blockly.Categories.control);
```

Comment:
Did you mean to remove setting the category?

Reply:
Yep, that's exactly why :)

Reply:
Is this fixed in a follow up or should we have an issue so we don't lose it?

Reply:
It's just a no-op introduced in https://github.com/scratchfoundation/scratch-blocks/commit/d7745b12fdb5ee9a4761b7e99fd201673e0e7a8e AFAICT; the comment indicates it's for styling, and the blocks style fine without it.

Reply:
Hmm it looks like it's used for surfacing some data in the dom as well. https://github.com/search?q=repo%3Ascratchfoundation%2Fscratch-blocks+data-category&type=code

And is also used for note playing for some reason?? https://github.com/scratchfoundation/scratch-blocks/blob/1e928c6e75b944ab73e93f589ed30c87e7e86f74/core/field_note.js#L693

Reply:
I don't see anything that actually reads or otherwise manipulates that data attribute though, it just gets set in those places. The field note thing is interesting, but I have that working in a forthcoming PR.

Reply:
Ok I'm satisfied with that! I think we've done due diligence and I don't think anyone in the world is gonna have more context.

Reply:
Sounds good, and thanks for double checking on this!